pkg/controller/deactivation/deactivation_controller_test.go

```diff
@@ -314,3 +418,8 @@ func assertThatUserSignupDeactivated(t *testing.T, cl *test.FakeClient, name str
 	require.NoError(t, err)
 	require.Equal(t, expected, userSignup.Spec.Deactivated)
 }
+
+func newHostOperatorConfigWithReset(t *testing.T, options ...test.HostOperatorConfigOption) *toolchainv1alpha1.HostOperatorConfig {
+	t.Cleanup(hostoperatorconfig.Reset)
+	return test.NewHostOperatorConfig(options...)
+}
```

Comment:
btw, there is already one function doing the very same thing https://github.com/codeready-toolchain/host-operator/blob/master/test/config.go#L11-L14 is there any reason for adding it here?